test/k8s-integration/config/test-config-template.in

```diff
@@ -4,6 +4,11 @@ StorageClass:
 SnapshotClass:
   FromFile: {{ .SnapshotClassFile }}
 {{end}}
+{{if .Timeouts}}
```

Comment:
These timeouts are no longer needed since we're just skipping the tests?

Reply:
removed